website/community/2-resources.mdx

```diff
@@ -68,6 +68,7 @@ See the <a href={require('@docusaurus/useBaseUrl').default('showcase')}>showcase
 - [docusaurus-plugin-typedoc](https://github.com/tgreyuk/typedoc-plugin-markdown/tree/master/packages/docusaurus-plugin-typedoc) - A Docusaurus plugin to build documentation with [TypeDoc](https://typedoc.org/)
 - [docusaurus-openapi-docs](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs) - A Docusaurus plugin and theme for generating interactive OpenAPI docs
 - [docusaurus-post-generator](https://github.com/moojing/docusaurus-post-generator) - A command line tool for user to add a blog/doc file quickly by command like `yarn gen-post new [template] [post_name]`.
+- [docusaurus-graph](https://github.com/Arsero/docusaurus-graph) - A Docusaurus plugin for generating a graphview from your documentation files. The graphview shows the links between files.
```

Comment:
```suggestion
- [docusaurus-graph](https://github.com/Arsero/docusaurus-graph) - A Docusaurus plugin for generating a graphview from your documentation files. The graph view shows the links between files.
```

Reply:
I have a question, why the word "graphview" pass in the first sentence ? 😂